plugin.rb

```diff
@@ -38,6 +38,8 @@ module ::DiscourseDataExplorer
   require_relative "lib/discourse_data_explorer/queries"
   require_relative "lib/discourse_data_explorer/query_group_bookmarkable"
 
+  GlobalSetting.add_default(:max_data_explorer_api_reqs_per_10_seconds, 2)
```

Comment:
Need some validation whether this is a good way to add a global setting from a plugin. I verified that this works, and can be overridden e.g. from `discourse_defaults.conf`, so functionally it does the trick.

Reply:
As noted on the call, yes it's all good:

https://github.com/discourse/discourse/blob/3a9a534881135ac8928266b1d02bc679f2b49b31/plugins/chat/plugin.rb#L28-L28